consvc_shepherd/management/commands/sync_boostr_data.py

```
@@ -47,10 +39,14 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         """Handle running the command"""
+        env = environ.Env()
+        BASE_DIR = Path(__file__).resolve().parent.parent
+        environ.Env.read_env(BASE_DIR / ".env")
+
         loader = BoostrLoader(
             options["base_url"],
-            options["email"],
-            options["password"],
+            env("BOOSTR_API_EMAIL"),
```

@cyptm can we think about what will happen if those values are not provided in the env?

What I liked about the previous version is that the args were required and the script would not run without them, so we'd fail fast and let the caller know there is a problem.

Do you know if the `env` function on line 48 will raise an exception if the value is empty or missing? If not, I think we would get all the way to the point of making the authentication request to Boostr and then getting an "unauthorized" response before the issue of missing values would cause failure.

If `.env` doesn't do it for us, could we adjust this code to fail fast if the env vars are not provided? Like maybe line 45, after we read the env but before we try to initialize the BoostrLoader with missing values?

---

Lovely, thanks for checking on that! :shipit: